### PR TITLE
npm install react-router-dom@6, created Pages folder and three page c…

### PR DESCRIPTION
**Pull request by MartynasKupsys**
…omponents, setup routes, tested

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "no-use-before-define": "off",
     "jsx-a11y/media-has-caption": "off",
     "jsx-a11y/no-noninteractive-element-interactions": "off",
-    "react/jsx-filename-extension": ["warn", { "extensions": [".jsx", ".tsx"] }],
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js",".jsx", ".tsx"] }],
```
Comment (Valdas-Stonkus) [inline .eslintrc.json:29]: Today eslint was fixed, check for dublicates
https://github.com/prodevshub/community-blog/commit/c2e82911bc90c8554fdd6d7cc9e8bf802fc8b30e

**Review by Valdas-Stonkus — APPROVED** [review]
Good job.

**Review by suiraDN — APPROVED** [review]
👍🏻

**Review by INKR1 — APPROVED** [review]

**Review by MarcinkEve — APPROVED** [review]

**Comment by MarcinkEve** [comment]
Nice 👍🏻

**Review by prodevshub — APPROVED** [review]

**Pull request merged**